src/renderer/constants.js

```diff
@@ -22,9 +22,10 @@ const formats = {
 };
 
 const intervals = {
+  HOLDINGS_POLLING: 60 * 1000,
```

Comment:
Since we are already reducing the RPC calls to get transaction details with these changes, we probably don't need to reduce this polling rate as much; what do you think about increasing this from 15 seconds to 30 seconds instead of increasing to 60.